jobs/director/spec

```diff
@@ -429,10 +429,15 @@ properties:
 
   # Compiled Package Cache
   compiled_package_cache.provider:
-    description: Provider of the blobstore used for the compiled package cache
+    description: |
+      Provider of the blobstore used for the compiled package cache.
+      Possible values are: 'local', 's3' and 'gcs'.
+      A value of 'local' is implied (and thus optional) when blobstore_path is provided.
     default: 's3'
   compiled_package_cache.options.bucket_name:
-    description: AWS S3 Bucket used for the compiled package cache
+    description: AWS S3 or GCS Bucket used for the compiled package cache
+  compiled_package_cache.options.blobstore_path:
```

Comment:
Why was this field added? It seems to go beyond improving the description of job properties? Not saying it is wrong just lacking context here.

Reply:
Yes, this property was already existing and supported, but missing here in the spec, and thus not documented.

It has been added here because it has an impact on `compiled_package_cache.provider`. So, it's mentionned at line 435 above: “_A value of 'local' is implied (and thus optional) when blobstore_path is provided._”

This explains why`compiled_package_cache.options.blobstore_path` has been documented here.